addons/todo/static/src/views/todo_form/todo_form_controller.js

```diff
@@ -1,14 +1,55 @@
 /** @odoo-module **/
 
+import { _lt } from "@web/core/l10n/translation";
 import { FormController } from "@web/views/form/form_controller";
 import { TodoEditableBreadcrumbName } from "@todo/components/todo_editable_breadcrumb_name/todo_editable_breadcrumb_name";
+const { onWillStart } = owl;
 
 /**
  *  The FormController is overridden to be able to manage the edition of the name of a to-do directly
  *  in the breadcrumb.
  */
 
-export class TodoFormController extends FormController {}
+export class TodoFormController extends FormController {
+    setup() {
+        super.setup();
+        onWillStart(async () => {
+            this.conversionViewId = await this.model.orm.call(
+                'project.task',
+                'get_conversion_view_id',
+                [],
+            );
+        });
+    }
+
+    get actionMenuItems() {
+        this.archiveEnabled = true;
+        const actionToKeep = ["archive", "unarchive", "duplicate", "delete"];
+        const printToKeep = [];
+        const menuItems = super.actionMenuItems;
+        const filteredActions = menuItems.action?.filter(action => actionToKeep.includes(action['key'])) || [];
+        const filteredPrints = menuItems.print?.filter(print => printToKeep.includes(print['key'])) || [];;
```

Comment:
Moreover, since `printToKeep` is a empty array we could just defined an empty array instead of doing a filter, right? 🤔 

If yes then we could directly do that: 
```suggestion
        menuItems.print = [];
```

and remove `printToKeep` variable. 🙂